typescript/ecs/ecs-service-with-task-networking/index.ts

```diff
@@ -0,0 +1,49 @@
+import cdk = require('@aws-cdk/cdk');
+import ec2 = require('@aws-cdk/aws-ec2');
+import ecs = require('@aws-cdk/aws-ecs');
+import { Protocol } from '@aws-cdk/aws-ecs';
+
+// Based on https://aws.amazon.com/blogs/compute/introducing-cloud-native-networking-for-ecs-containers/
+class Ec2ServiceWithTaskNetworking extends cdk.Stack {
```

Comment:
I like the idea of demonstrating that these can be wrapped up in a class, but for purposes of still allowing more configurability, I'm wondering if it might be better to still keep the independent components separate as we do in some of the other ecs service examples?

Reply:
Done